src/controllers/AuthController.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-unused-vars */
+import token from 'uuid';
+import model from '../models';
+import {
+  validate, inValidName, inValidEmail, inValidPassword, magicTrimmer
+} from '../utils/validator';
+import { sendErrorResponse, sendSuccessResponse } from '../utils/sendResponse';
+import { hashPassword, comparePassword } from '../utils/passwordHash';
+import { SendMail, sendForgotPasswordMail } from '../services/emailsender';
+import { createToken } from '../utils/processToken';
+
+const { User } = model;
+
+// returns a token for logged in user or during sign up
+const userToken = (user) => {
+  const {
+    email, name, role, uuid
+  } = user;
+  return {
+    token: createToken({
+      uuid,
+      name,
```

Comment:
you destructured "name" from the user model but no name is defined there.

Reply:
Fixed.